modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/db2.sql

```diff
@@ -24,3 +24,12 @@ CREATE INDEX IDX_USER_ID ON IDN_AUTH_USER_SESSION_MAPPING (USER_ID)
 
 CREATE INDEX IDX_SESSION_ID ON IDN_AUTH_USER_SESSION_MAPPING (SESSION_ID)
 /
+
+CREATE INDEX IDX_OCA_UM_TID_UD_APN ON IDN_OAUTH_CONSUMER_APPS(USERNAME,TENANT_ID,USER_DOMAIN, APP_NAME)
```

Comment:
We need to create the indexes if they don't already exist

Reply:
https://www.ibm.com/support/knowledgecenter/en/SSEPGG_11.1.0/com.ibm.db2.luw.sql.ref.doc/doc/r0000919.html

This is handling by default in db2 for other db types  changes Addressed